<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="findRadioBtn">
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="replaceRadioBtn">
       <property name="text">
        <string>替换</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>替换为：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="matchCaseCheck">
         <property name="text">
          <string>区分大小写</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="findRepeatCheck">
         <property name="text">
          <string>循环查找</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="matchWholeWordCheck">
         <property name="text">
          <string>全词匹配</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="findContentCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QPushButton" name="replaceAllBtn">
       <property name="text">
        <string>全部替换</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>查找内容：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="findNextBtn">
       <property name="text">
        <string>查找下一个</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/res/Image/next.png</normaloff>:/res/Image/next.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="replaceCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QPushButton" name="replaceBtn">
       <property name="text">
        <string>替换</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QPushButton" name="replaceSelectionBtn">
       <property name="text">
        <string>在选区中替换</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="findPreviousBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/res/Image/previous.png</normaloff>:/res/Image/previous.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" rowspan="2" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>查找模式</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="modeNormalRadio">
          <property name="text">
           <string>普通</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="modeExtionsionRadio">
          <property name="text">
           <string>扩展(\n, \r, \t, \0, \x...)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="modeRegExpRadio">
          <property name="text">
           <string>正则表达式</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
